shell/common/shell.h

```diff
@@ -298,6 +298,28 @@ class Shell final : public PlatformView::Delegate,
   ///
   bool IsSetup() const;
 
+  /// @brief  Add a non-implicit view.
```

Comment:
I am actually not sure what the style is, but the docs of the next two methods all use this grammatical form:

```suggestion
  /// @brief  Adds a non-implicit view.
```